react/week2/todolistsapp/src/components/TodoListItem.jsx

```diff
@@ -0,0 +1,22 @@
+export default function TodoListItem (props) {
+    const {todo, id, deleteTodo} = props;
+
+    function markTodo(todoId) {
+        const labels = document.querySelectorAll('label');
+        
+        for (const label of labels) {
+            if(Number(label.id) === todoId)
+                (label.children.item(0).checked) ?  label.style.textDecoration = "line-through" :  label.style.textDecoration = "none"
+        }
```

Comment:
We almost never use DOM in React and we try to use React itself, or if we really need to keep track of an DOM element we use `React.createRef()` and pass it as a ref to an element.
Here you are only changing the interface with CSS and you are not really updating your todos data (making it done or not done), The better way is to have this in your todo item and change that once the user click on the checkbox, so the todo item will look something like this:
`{ id: 1, description: 'Get out of bed', completed: true }`,
then you can have a function so it would update that specific todo item. That's more likely how it would be in the real life scenario.